### ISSUE TITLE
Missing connection.remoteAddress/remotePort

### ISSUE THREAD

**Issue by fed135** [mafintosh/turbo-net#4]
In order to maintain API compatibility with the core `net` module, connections would need to expose the `remoteAddress` and `remotePort` proprieties.

**Comment by LuKks** [comment]
I tried do this but I don't get it at well how works C and that API.
Any idea? I'll make the rest of JavaScript.

**Comment by aichholzer** [comment]
@LuKks -Your [PR](https://github.com/mafintosh/turbo-net/pull/15) does not include any C code changes?
I will try to get a PR in today that does the C part, but can't make any promises... Too much going on at the moment.

**Comment by LuKks** [comment]
Exactly, no C changes because it's just a timer, same functionality than "net" module.

**Comment by LuKks** [comment]
Would be perfect have the remote address, it's essential. I think that most application use it.
Surely I can do the JavaScript part, I had tried the part of C but I was not able.

**Comment by LuKks** [comment]
https://github.com/mafintosh/turbo-net/pull/15
